docs/README.md

```diff
@@ -1,12 +1,24 @@
 # Kremling
-Kremling is an npm library for writing css with React. It uses [React hooks](https://reactjs.org/hooks), is only 4kb gzipped,
-and requires no changes to your webpack config or build process.
+Kremling is an npm library for writing CSS with React. It is only 4kb gzipped, and requires no changes to your webpack config or build process.
 
 [Kremling Github page](https://github.com/CanopyTax/kremling)
 
 [Walkthrough](/walkthrough/getting-started.md)
 
-## Hooks example
+## Motivation
+Why use Kremling over say, CSS Modules?
```

Comment:
I would rather not call out specific competitor libraries and try to justify why kremling is better than them (since I honestly don't really believe that kremling is actually better than every other css lib option). Instead, I'd rather just focus on why we built kremling.

So I'd prefer to remove this sentence that calls out CSS modules